services/web/server/src/simcore_service_webserver/activity/handlers.py

```diff
@@ -0,0 +1,93 @@
+
+        cpu_query = 'irate(container_cpu_usage_seconds_total{container_label_node_id=~".+", container_label_user_id="' + str(user_id) + '"}[30s]) * 100'
+        memory_query = 'container_memory_usage_bytes{container_label_node_id=~".+", container_label_user_id="' + str(user_id) + '"} / 1000000'
+        just_a_metric = 'container_cpu_user_seconds_total{container_label_node_id=~".+", container_label_user_id="' + str(user_id) + '"}'
+        
+        config = request.app['servicelib.application_keys.config']['activity']
+        url = URL(config.get('prometheus_host')).with_port(config.get('prometheus_port')).with_path('api/' + config.get('prometheus_api_version') + '/query')
+
+        async def get_cpu_usage():
+            async with session.get(url.with_query(query=cpu_query)) as resp:
+                result = await resp.json()
+                return result
+
+        async def get_memory_usage():
+            async with session.get(url.with_query(query=memory_query)) as resp:
+                result = await resp.json()
+                return result
+
+        async def get_celery_reserved():
+            return get_celery(request.app).control.inspect().reserved()
+
+        async def get_container_metric_for_labels():
+            async with session.get(url.with_query(query=just_a_metric)) as resp:
+                result = await resp.json()
+                return result
+
+        results = await asyncio.gather(get_cpu_usage(), get_memory_usage(), get_celery_reserved(), get_container_metric_for_labels())
+        cpu_usage = results[0]['data']['result']
+        mem_usage = results[1]['data']['result']
+        metric = results[3]['data']['result']
+        celery_inspect = results[2]
+
+        res = {}
+        for node in cpu_usage:
+            node_id = node['metric']['container_label_node_id']
+            usage = node['value'][1]
+            res[node_id] = {
+                'stats': {
+                    'cpuUsage': usage
+                }
+            }
+
+        for node in mem_usage:
+            node_id = node['metric']['container_label_node_id']
+            usage = node['value'][1]
+            if node_id in res:
+                res[node_id]['stats']['memUsage'] = usage
+            else:
+                res[node_id] = {
+                    'stats': {
+                        'memUsage': usage
+                    }
+                }
+
+        for node in metric:
+            limits = {
+                'cpus': 0,
+                'mem': 0
+            }
+            metric_labels = node['metric']
+            limits['cpus'] = float(metric_labels.get('container_label_nano_cpus', 0)) / pow(10, 9) # Nanocpus to cpus
+            limits['mem'] = float(metric_labels.get('container_label_mem_limit', 0)) / pow(1024, 2) # In MB
+            node_id = metric_labels.get('container_label_node_id')
+            res[node_id]['limits'] = limits
+
+
+        for dummy_worker_id, worker in celery_inspect.items():
```

Comment:
FYI: if you do not use a variable you can do as 
```python
for _, worker in celery_inspect.items():
  # ...

```

Reply:
> FYI: if you do not use a variable you can do as
> 
> ```python
> for _, worker in celery_inspect.items():
>   # ...
> ```

Yeah! I used dummy because it was preferred for some users... I also prefer how '_' looks... but with dummy pylint doesn't complain either.